docs/snapshotting/snapshot-support.md

```diff
@@ -123,6 +123,9 @@ The baseline for snapshot resume latency target on Intel is under **8ms** with
   deal with cryptographic secrets. Please see [Snapshot security and uniqueness](#snapshot-security-and-uniqueness).
 - Snapshotting on arm64 works for both GICv2 and GICv3 enabled guests.
   However, restoring between different GIC version is not possible.
+- Running with the host kernel command line parameter `console=ttyAMA0` on
+  `aarch64` degrades the snapshot restore time to 8.5ms because the host kernel
+  logs generated in the restore process are slower with this configuration.
```

Comment:
This configuration leads to a degradation in performance in multiple scenarios, not only on snapshot restore. Would it be better to mention it in the `host-prod-setup` ?

Reply:
+1, I think it is correlated to the entry we have in the prod-host-setup on the serial device

Reply:
The statement is not completely correct as `ttyAMA0` is the name given to the serial console for specific ARM boards that have a specific serial controller, as seen [here](https://github.com/torvalds/linux/blob/v4.16/drivers/tty/serial/amba-pl011.c#L2488). Coincidence makes it that the controller that determines the name is quite popular and most ARM boards will have a `ttyAMA0` port.

A better way of explaining the problem would be to explain the actual effect and mitigation. I'm not 100% sure that this doesn't happen on x86 when enabling console output on `ttyS0`.

As @serban300 mentioned, I'd put this under the prod host setup chapter.